Add chat hotkey

eros71-dev opened the pull request:
Added a new key that let's you focus on the chat box, Y by default.
Preview below:
![imagen](https://user-images.githubusercontent.com/16540103/139517770-417f53b1-6de7-4629-9f66-2f87b17fb280.png)

eros71-dev committed:
Add chat hotkey

Added a new key that let's you focus on the chat box, Y by default.

justabluemouse commented:
I 100% love this idea, it's so annoying to have to click off the game onto the chat box in order to chat

eros71-dev committed:
Merge branch 'sm64js:MMO' into MMO

eros71-dev committed:
Site rewrite

[THIS UPDATE HASN'T BEEN FINISHED YET]
The site has been rewritten from the ground up to remove all external dependencies like bootstrap or jQuery.

The site now loads a lot faster, but some stuff is not working now:

 - The game canvas and chat don't load.
 - The rules modal/agreement doesn't work.
 - The pop ups to change the controls, Mario's colors, etc are probably broken.
 - And the Credits page won't load the css.

eros71-dev committed:
QoL Fixes, New logo animation

Everything related to external libraries is broken, same with the credits page (this last problem happened a while ago when the credits page didn't exist and I added it, but I forgot how that got fixed, it's css doesn't load, the page itself does)

eros71-dev commented:
@Agent-11 added it to their repo, closing pr request to avoid any duplicates or problems
